key.js

```diff
@@ -43,29 +286,29 @@ const clickSoundRed = () => {
 
 addEventListener('keydown', (e) => {
     const direction = e.key;
-    if (direction[5] == 'U') {
+    if (direction === 'ArrowUp') {
         dUp.style.transform = `scale(.95) translateY(-3px)`;
         dUp.style.boxShadow = `0px 0px 10px 5px black inset`;
         dUp.style.transition = `.3s ease all`;
```

Comment:
Anything that has to do with the DOM is supposed to be in the `view`, a `view` can have its own state.